cypress/integration/app-init.spec.js

```diff
@@ -0,0 +1,8 @@
+describe('App initialization', () => {
+  const baseUrl = Cypress.config().baseUrl;
+  it('Loads home page', () => {
+    cy.visit(baseUrl);
+
+    cy.get('div#root').should('exist');
```

Comment:
It's pretty useless test and even doesn't check that script loading, as div#root is in initial html before any rendering